src/commons/assessmentWorkspace/AssessmentWorkspace.tsx

```diff
@@ -472,9 +472,22 @@ const AssessmentWorkspace: React.FC<AssessmentWorkspaceProps> = props => {
             <SideContentContestLeaderboard
               handleContestEntryClick={handleContestEntryClick}
               orderedContestEntries={(question as IContestVotingQuestion)?.scoreLeaderboard ?? []}
+              leaderboardType={SideContentType.scoreLeaderboard}
             />
           ),
           id: SideContentType.scoreLeaderboard
+        },
+        {
+          label: 'Popular Vote Leaderboard',
+          iconName: IconNames.PEOPLE,
+          body: (
+            <SideContentContestLeaderboard
+              handleContestEntryClick={handleContestEntryClick}
+              orderedContestEntries={(question as IContestVotingQuestion)?.scoreLeaderboard ?? []}
```

Comment:
Is this supposed to be `popularVoteLeaderboard`?

Reply:
Yes, sorry about that. Will fix that.